### PR TITLE
Add support for graphic circle and field

### PR DESCRIPTION
**Pull request by fkryvyts-ingrid**

### CODE REVIEW

File: drawers/graphic_field.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+			}
+
+			width := field.RowBytes * 8
+			height := field.DataBytes / field.RowBytes
+
+			img := image.NewGray(image.Rect(0, 0, width, height))
+
+			for y := 0; y < height; y++ {
+				for x := 0; x < width; x++ {
+					val := ((field.Data[y*(width/8)+x/8]) >> (7 - x%8)) & 0xF
```
Comment (mknycha-ingrid) [inline drawers/graphic_field.go:31]: This line is hard to understand, could you move it to a function or at least add a comment?

File: parsers/graphic_field.go [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+)
+
+func NewGraphicFieldParser() *CommandParser {
+	const code = "^GF"
+
+	return &CommandParser{
+		CommandCode: code,
+		Parse: func(command string, printer *printers.VirtualPrinter) (interface{}, error) {
+			result := &elements.GraphicField{
+				Position: printer.NextElementPosition,
```
Comment (mknycha-ingrid) [inline parsers/graphic_field.go:20]: What about assigning the default compression type `A` here?

Comment (fkryvyts-ingrid) [reply parsers/graphic_field.go:20]: I thought about it but ZPL specification says that property field has no default value

Comment (mknycha-ingrid) [reply parsers/graphic_field.go:20]: Ok, could you point me to the documentation? I could not fetch the one on `zebra.com`, so I was referring to this one: https://www.servopack.de/support/zebra/ZPLII-Prog.pdf

Comment (fkryvyts-ingrid) [reply parsers/graphic_field.go:20]: Yes, sure, I mainly use hints that labelary provides
Screenshot from 2024-03-01 13-26-54

File: drawers/graphic_field.go (hunk 1 above)
Comment (fkryvyts-ingrid) [reply drawers/graphic_field.go:34]: @mknycha-ingrid Added comment, please check if it is better to understand this way

File: parsers/graphic_field.go (hunk 2 above)
Comment (mknycha-ingrid) [reply parsers/graphic_field.go:20]: ok

**Pull request merged**